InitializeRepo/initialize.sh

```
@@ -0,0 +1,34 @@
+#!/bin/bash
+
```

Add a `set -euo pipefail` here to fail fast if there any errors, especially to avoid making any weird irreversible changes later.

---

Added